Makefile

```diff
@@ -37,8 +46,8 @@ test: fmtcheck generate
 	VAULT_ACC=1 go test -tags='$(BUILD_TAGS)' $(TEST) -v $(TESTARGS) -timeout 45m
 
 # test-acceptance runs all acceptance tests
-test-acceptance:
-	bats $(CURDIR)/tests/acceptance/basic.bats
+test-acceptance: $(if $(WITH_DEV_PLUGIN), dev-acceptance)
+	bats -f $(TESTS_FILTER) $(CURDIR)/tests/acceptance/basic.bats
```

Comment:
Should we document the `WITH_DEV_PLUGIN` and `TESTS_FILTER` options in the README?

Reply:
Fixed in https://github.com/hashicorp/vault-plugin-secrets-azure/pull/85/commits/322a5da4372af9d3a6295372ed241f6ddebcb660